force_bdss/core_plugins/service_offers_plugin.py

```diff
@@ -0,0 +1,59 @@
+from envisage.api import ServiceOffer
+from traits.api import Instance, List, Interface
+
+from force_bdss.api import BaseExtensionPlugin
+
+
+class ServiceOffersPlugin(BaseExtensionPlugin):
+    """A plugin which is able to contribute one or more user-made UI objects
+    via the envisage ServiceOffer protocol. This plugin can handle
+    multiple types of UI objects, via appropriate configuration of the
+    object returned by get_service_offers"""
```

Comment:
Can we make this docstring generic (no mentions of UIs) and move all the UI-related remarks to the "Example" chapter of the `get_service_offer_factories` docstring? (and add an intro to the "Example" to mention that the example is showcasing a `wfmanager` extension functionality)

Reply:
Both suggestions should now be implemented